src/content/docs/discovery/index.mdx

```diff
@@ -29,12 +31,73 @@ The next step is to plan, build, optimize, and launch a production-ready storefr
 
 </Callouts>
 
+## Tips for a successful project
+
 The discovery phase is important because Adobe Commerce is a highly customizable platform with a large thid-party extension ecosystem. Migrating an Adobe Commerce storefront to Edge Delivery Services is similar to migrating to a headless storefront (like [PWA Studio](https://developer.adobe.com/commerce/pwa-studio)).
 
 :::note[Important]
 The key requirement for a headless storefront implementation is that all required data is provided by APIs.
 :::
 
+Successfully building a storefront requires a well-defined plan and a phased approach. Here are some key steps to consider:
+
+<Tasks>
+
+<Task>
+
+### Phased launch
+
+Fast return on investment (ROI) and learnings:
+
+* Break down project scope into small milestones
+* Small milestones result in faster time-to-value (TTV), learnings, and higher quality
+* Define a rollout _and_ rollback strategy for each milestone (for example, split traffic)
```

Comment:
Can we provide more detailed guidance on what a rollback looks like?

Reply:
"Having a rollback" strategy typically means that when you do the migration from Commerce to Edge Delivery that you avoid doing any potentially breaking changes.

For example if you uninstall an extension during the migration, because it's not needed on Edge but needed on Luma. Then this is a breaking change which would prevent you from rolling back to Luma once on Edge.

To actually do the switch customers can use Fastly VCLs, DNS switches or traffic splits / A/B testing. We have seen all of those in projects, all of the methods work fine.